### PR TITLE
Version change and minor edits

### PR DESCRIPTION
**Pull request by cheenamalhotra**

### CODE REVIEW

File: contributing-workflow.md [hunk 1]
```diff
@@ -80,8 +90,26 @@ The package naming conventions follow SemVer 2.0.0 and also provide changeset in
...
-- `1.1.0-build` - Identitier for currently active driver version in Build.
+- `1.1.0-build` - Identifier for currently active driver version in Build.
 - `19250.1` - Unique identifier to keep latest PRs on top of the feed.
 - `c21aa7c` - Short Commit Id to identify merged commit in `master`.
 
 > Note: This public feed is only for testing and validating new PR changes. Packages from feed will be eventually removed when the maximum NuGet Package limit of **50** is reached. We do not recommend using packages from this feed in client applications.
+
+## Nightly builds
+
+We also maintain NuGet feed for Nightly builds that are generated when new changes are merged in the repository. To access Nightly builds, add below configuration to package sources:
```
Comment (JRahnama) [inline contributing-workflow.md:101]: NIT: should we change it to `nightly build` with lower case ?

**Comment by ErikEJ** [comment]
So it is now possible to use daily builds, and the roadmap for 3.0 is available?

File: contributing-workflow.md (hunk 1 above)
Comment (cheenamalhotra) [reply contributing-workflow.md:101]: Headings are often sentence case.

**Comment by cheenamalhotra** [comment]
@ErikEJ Yes for Nightly builds, for RoadMap - planning is in progress :)

File: buildAddons.cmd [hunk 2]
```diff
@@ -2,6 +2,7 @@ call :pauseOnError msbuild /p:configuration=Release /t:clean
 call :pauseOnError msbuild /p:configuration=Release /t:BuildAllConfigurations
 call :pauseOnError msbuild /p:configuration=Release /t:BuildAKVNetFx
 call :pauseOnError msbuild /p:configuration=Release /t:BuildAKVNetCoreAllOS
+call :pauseOnError msbuild /p:configuration=Release /t:BuildAKVNetStandardAllOS
```
Comment (karinazhou) [inline buildAddons.cmd:5]: I find `BuildAKVNetStAllOS` in build.proj . Can we keep the consistency?

Comment (cheenamalhotra) [reply buildAddons.cmd:5]: My typo

**Review by karinazhou — APPROVED** [review]

**Review by JRahnama — APPROVED** [review]

**Review by johnnypham — APPROVED** [review]

**Pull request merged**